src/components/questionStructure/Task/index.tsx

```diff
@@ -18,7 +25,8 @@ export type TaskProps =
   | CodeTaskProps
   | MCQOneTaskProps
   | MCQMultiTaskProps
-  | HandwritingTaskProps
+  | MathsProcessedHandwriting
```

Comment:
For naming consistency:

```suggestion
  | ProcessedHandwritingProps
```

Reply:
Resolved in c17cf61852daf220a4a54d1dd8be120b5687cf44